### ISSUE TITLE
[Bug] Module not found: Can't resolve 'aptos_dynamic_transaction_composer_bg.wasm'

### ISSUE THREAD

**Issue by RosalinaLuma620** [aptos-labs/aptos-ts-sdk#573]
# 🐛 Bug Description

When I delete the yarn.lock file ,and rebuild the project which I was developing ,

![image](https://github.com/user-attachments/assets/a421d44b-a2c0-45c1-8e5b-1463def0361c)

I got this error. It turns out somehow the SDK dependency needs "@wgb5445".

Import trace for requested module:
./node_modules/@aptos-labs/ts-sdk/dist/esm/chunk-4PBXTMN3.mjs
./node_modules/@aptos-labs/ts-sdk/dist/esm/index.mjs
./pages/_app.tsx
 ⨯ ./node_modules/@wgb5445/aptos-intent-npm/dist/esm/main.mjs:1176:0
Module not found: Can't resolve 'aptos_dynamic_transaction_composer_bg.wasm'

Did you guys change this SDK ""@aptos-labs/ts-sdk" Dependency ? Or did I do something wrong .?

**Comment by WGB5445** [comment]
Sorry for the inconvenience caused. Recently, we made some changes to introduce new features, aiming to build trading scripts using TypeScript. This is a bold experiment, and so far, we haven't encountered this issue in our current testing.

I would like to help resolve the issue. Could you please provide more details about your environment? Thank you very much!

**Comment by WGB5445** [comment]
Since Next.js makes some data requests during the build process, but this particular file doesn't actually need to make any requests in this build, we have two solutions:

- Use the old version of the ts-sdk and lock the version.
- Use a configuration like the one below during the Next.js build process.

```
import type { NextConfig } from "next";
import { webpack } from "next/dist/compiled/webpack/webpack";

const nextConfig: NextConfig = {
  /* config options here */
  webpack (config, { isServer }){
    config.plugins.push(
      new webpack.IgnorePlugin({
        resourceRegExp: /aptos_dynamic_transaction_composer_bg.wasm/,
      })
    );
    return config;
  }
};

export default nextConfig;

```

**Comment by WGB5445** [comment]
We expect to fully fix this issue in a future release. 

Thank you for your feedback, and if you have any further questions, feel free to ask. 

We'll be here to help!

**Comment by 0xmaayan** [comment]
Hi @RosalinaLuma620 , thanks for reporting! We have released a new version with a fix `1.32.1`
https://www.npmjs.com/package/@aptos-labs/ts-sdk

**Comment by RosalinaLuma620** [comment]
> Sorry for the inconvenience caused. Recently, we made some changes to introduce new features, aiming to build trading scripts using TypeScript. This is a bold experiment, and so far, we haven't encountered this issue in our current testing.
> 
> I would like to help resolve the issue. Could you please provide more details about your environment? Thank you very much!

Thanks a lot , my environment is Mac OS M2, Node version : v20.10.0 , Next.js V13.3.1 , npm version 10.2.3 , yarn version 1.22.22

**Comment by RosalinaLuma620** [comment]
> Since Next.js makes some data requests during the build process, but this particular file doesn't actually need to make any requests in this build, we have two solutions:
> 
> * (currently better) Use the old version of the ts-sdk and lock the version.
> * Use a configuration like the one below during the Next.js build process.

Yes , I've use the original yarn.lock file to rebuild this project , it's work .

**Comment by RosalinaLuma620** [comment]
> Sorry for the inconvenience caused. Recently, we made some changes to introduce new features, aiming to build trading scripts using TypeScript. This is a bold experiment, and so far, we haven't encountered this issue in our current testing.

Yeap , I know that , it's huge . To the moon , respect ~

**Comment by RosalinaLuma620** [comment]
> Hi @RosalinaLuma620 , thanks for reporting! We have released a new version with a fix `1.32.1` https://www.npmjs.com/package/@aptos-labs/ts-sdk

Thanks a lot . :)
